examples/browser-webrtc/client/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+use async_channel::bounded;
```

Comment:
Why not use `futures::channel`?

Reply:
That's just me being more familiar with `async-channel` as I find the `futures::channel` docs not as good. 

BUT, I have overcome this and learned all about `futures::channel` and the implementation is now switched 😁